fineract-e2e-tests-core/src/test/java/org/apache/fineract/test/api/ApiProperties.java

```diff
@@ -32,6 +32,8 @@ public class ApiProperties {
     private String username;
     @Value("${fineract-test.api.password}")
     private String password;
+    @Value("${fineract-test.api.valid-password}")
```

Comment:
What is "valid password" means? Why do we need this one? why the "password" field is changed to this every places?

Reply:
E2E tests create users for  testing purpose. After I switched password validation policy the "password" value is no longer valid when creating new users or changing the passwords.